source/anaconda/index.rst

```diff
@@ -0,0 +1,95 @@
+Anaconda
+=========
+
+Anaconda 是一个 用于科学计算的 Python 发行版，支持 Linux、macOS 和 Windows。
+其提供了方便使用的包管理器和环境管理器。
+
+**建议所有 Python用户使用 Anaconda 而非 Linux 或 macOS 系统自带的 Python。**
+
+安装
+----
+
+1. 下载 Anaconda
+
+   进入 `Anaconda 官方下载页面 <https://www.anaconda.com/products/individual#Downloads>`__，
+   会看到类似下图的下载页面。根据自己的系统选择对应的安装包（通常选择下图中红框圈出的安装包）。
+
+   .. image:: anaconda-download.png
+
+   若官方下载速度较慢，可以直接从清华大学 Anaconda 镜像下载。
+   访问 https://mirrors.tuna.tsinghua.edu.cn/anaconda/archive/，找到文件名类似
+   :file:`Anaconda3-2020.11-Linux-x86_64.sh` 或
+   :file:`Anaconda3-2020.11-MacOSX-x86_64.sh` 的文件。
+   其中 ``2020.11`` 为 Anaconda 版本的发行日期，选择最新版本即可。
+
+2. 安装 Anaconda
+
+   Windows 用户直接双击安装包即可安装。
+
+   Linux 用户在终端执行如下命令以安装 Anaconda::
+
+      $ bash Anaconda3-2020.11-Linux-x86_64.sh -b
+
+   macOS 用户再终端执行如下命令以安装 Anaconda::
+
+      $ bash Anaconda3-2020.11-MacOSX-x86_64.sh -b
```

Comment:
What's the meaning of `-b` option?

Reply:
batch mode. No stopping for user input

Reply:
You mean all the options will use the default values?

Reply:
Yes